BilparkApp/src/components/LoginForm.js

```diff
@@ -0,0 +1,86 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { StyleSheet, Text, TextInput, TouchableOpacity, View } from 'react-native';
+import { reduxForm, Field } from 'redux-form';
+import { postUser, postCurrent } from '../actions/auth';
+
+const renderInput = ({ placeholder, input: { onChange, ...restInput } }) => {
+  return (
+    <TextInput
+      style={styles.input}
+      onChangeText={onChange}
+      {...restInput}
+      placeholder={placeholder}
+    />
+  );
+};
+
+class LoginForm extends Component {
+  constructor(props) {
+    super(props);
+    this.submit = this.submit.bind(this);
+    this.props.postCurrent();
+  }
+
+  submit(values) {
+    this.props.postUser(values.username, values.password);
+  }
+
+  render() {
+    const { handleSubmit } = this.props;
+
+    return (
+      <View style={styles.container}>
+        <Text>Skriv inn epost og passord:</Text>
+        <Field name="username" component={renderInput} placeholder="email" />
+        <Field name="password" component={renderInput} placeholder="password" />
```

Comment:
Make it so that the password field has the characters hidden.

Reply:
Done